Fix(Serverless): Add Explanation Regarding Fragmented Distributed Traces (#7082)
newrelic/docs-website: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx

Your telemetry will not appear under New Relic Serverless. Instead, you will find your telemetry data under the New Relic OpenTelemetry Nerdlets.

## Distributed Tracing

> In some cases you may see fragmented distributed traces within New Relic with this configuration. This occurs when a trace starts or involves a service that is outside the ADOT context. When this happens, trace spans are usually created by AWS X-Ray and ADOT is not aware of them beyond a trace context being passed to it. This most commonly happens with managed AWS services that ADOT may interact, but that can only be traced using AWS X-Ray.

Reviewer: ```suggestion
In some cases you may see fragmented distributed traces within New Relic with this configuration. This occurs when a trace starts or involves a service that is outside the ADOT context (for example, a managed AWS service). Spans from that service are created by X-Ray, not OpenTelemetry, and ADOT does not forward them to New Relic.
```

Reviewer: I'd also add something like "Although the traces appear fragmented, they still provide complete insight into performance within the serverless function as well as other services whose spans were forwarded to New Relic."

> New Relic has an [X-Ray integration](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-x-ray-monitoring-integration/) to capture these spans created by X-Ray outside of an ADOT context. However, it has not yet been updated to support OpenTelemetry. Once this integration is updated, you should be able to see a full distributed trace within New Relic using OpenTelemetry.

Reviewer: I don't think we should mention the X-Ray integration at all, unless it's to recommend using it without any OpenTelemetry configuration if the fragmented traces are a real blocker for you.

Author: Without that this is pretty much a dead end.

Reviewer: I totally agree we should prioritize the solution to forward and view _all_ AWS spans in New Relic. But as a general principle I don't think we should use language in the docs that promises particular implementations.
